cmd/editorconfig-checker/main.go

```diff
@@ -60,10 +61,15 @@ func init() {
 		configPaths = append(configPaths, defaultConfigFileNames[:]...)
 	} else {
 		configPaths = append(configPaths, configFilePath)
+
 	}
 
 	currentConfig, _ = config.NewConfig(configPaths)
 
+	if strings.Contains(currentConfig.Path, ".ecrc") {
```

Comment:
Please look at

https://github.com/editorconfig-checker/editorconfig-checker/pull/389#discussion_r1797617246

```suggestion
	if configFile Path == "" && strings.Contains(currentConfig.Path, ".ecrc") {
```

Also, please consider using hassuffix

```suggestion
	if configFile Path == "" && strings.HasSuffix(currentConfig.Path, ".ecrc") {
```

Reply:
I reopened the comment

Because I didn't see a fix for the point I raised

I commented here

https://github.com/editorconfig-checker/editorconfig-checker/pull/389#discussion_r1798291169